### PR TITLE
Added basic authorization

### PR DESCRIPTION
**Pull request by akashsinha13**
1. Removed CORS filter
2. Added very basic authorization

### CODE REVIEW

File: src/main/java/com/ekart/config/SecurityConfig.java [hunk 1]
```diff
@@ -50,12 +51,12 @@ protected void configure(HttpSecurity http) throws Exception {
 
         http.authorizeRequests()
                 .antMatchers("/login").permitAll()
-//                .antMatchers(HttpMethod.DELETE).hasRole("ADMIN")
-//                .antMatchers("/api/v1/brands/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers("/api/v1/category/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers("/api/v1/sizes/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers(HttpMethod.POST, "/api/v1/products/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers(HttpMethod.PUT, "/api/v1/products/**").hasAnyRole("ADMIN", "STAFF")
+                .antMatchers(HttpMethod.GET).permitAll()
```
Comment (prap12) [inline src/main/java/com/ekart/config/SecurityConfig.java:54]: i'm changing the /products Get method to Post due to same common method for all filters and sort conditions

File: src/main/java/com/ekart/config/SecurityConfig.java [hunk 2]
```diff
@@ -72,9 +73,9 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
     CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
         configuration.setAllowedOrigins(Collections.singletonList("http://localhost:4200/"));
```
Comment (prap12) [inline src/main/java/com/ekart/config/SecurityConfig.java:75]: remove the hardcoded origin part

Comment (akashsinha13) [reply src/main/java/com/ekart/config/SecurityConfig.java:75]: @prap12 , I tried using origin but it is not working..

Screenshot 2021-12-22 at 9 03 16 PM

also in request header, origin is present..

File: src/main/java/com/ekart/config/SecurityConfig.java (hunk 1 above)
Comment (akashsinha13) [reply src/main/java/com/ekart/config/SecurityConfig.java:54]: I have added this

File: src/main/java/com/ekart/config/SecurityConfig.java (hunk 2 above)
Comment (prap12) [reply src/main/java/com/ekart/config/SecurityConfig.java]: okay will check

**Review by prap12 — APPROVED** [review]

**Pull request merged**